module/os/freebsd/zfs/sysctl_os.c

```diff
@@ -381,18 +419,15 @@ SYSCTL_INT(_vfs_zfs, OID_AUTO, default_ibs, CTLFLAG_RWTUN,
  * around the disk. So a sane default for the space map block size
  * is 8~16K.
  */
-extern int zfs_metaslab_sm_blksz_no_log;
 SYSCTL_INT(_vfs_zfs_metaslab, OID_AUTO, sm_blksz_no_log, CTLFLAG_RDTUN,
 	&zfs_metaslab_sm_blksz_no_log, 0,
 	"Block size for space map in pools with log space map disabled.  "
 	"Power of 2 greater than 4096.");
-
```

Comment:
I can't quickly find it in style(9), but traditionally there should be the empty line before comment to accentuate the block that is commented.